packages/docker/Dockerfile.bionic

```diff
@@ -84,4 +85,24 @@ RUN echo 'buildbot ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
 
 USER buildbot
 
-ENTRYPOINT /entrypoint.sh "$BUILDMASTER" "$WORKERNAME" "$WORKERPASS"
+ENTRYPOINT /entrypoint.sh $BUILDMASTER "$WORKERNAME" "$WORKERPASS"
+
+#################################################################################################
+
+FROM base AS appimage
+
+COPY . /home/builder/src
+
+WORKDIR /home/builder/src
+
+RUN python setup.py -V
+
+RUN python setup.py sdist
```

Comment:
I would put the dpkg lines together, and the appimage lines also together. Like
```
RUN sdist
RUN bdist
RUN dpkg-deb

RUN app image
RUN out/appimage -t
```

Reply:
```
RUN python setup.py sdist
RUN python setup.py --command-packages=stdeb.command bdist_deb
RUN dpkg-deb -I deb_dist/pybitmessage_0.6.3.2-1_amd64.deb

RUN buildscripts/appimage.sh
RUN out/PyBitmessage-0.6.3.2.glibc2.15-x86_64.AppImage --appimage-extract-and-run -t
```
Like this?

Reply:
yes